app/views/mentor/chat_groups/show.html.haml

```diff
@@ -0,0 +1,8 @@
+-# .message
```

Comment:
__[HAML-Lint]__

3 consecutive comments can be merged into one


<!-- Comment by [SideCI](https://sideci.com) -->